.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,93 @@
+
+- The issue list of this repo is exclusively for bug reports and feature requests. Please come to the discord with general questions you may have.
+
+- If you have a question, you can get quick answers from the [Discord chat](https://discord.gg/dAaz472mPz).
+
+- If your issue is resolved but still open, please close it. In case you found a solution by yourself, it could be helpful to explain how you fixed it.
+
+- Generally, there are many features still remaining to produce a valid 1040 for most people. We hope you will view this project in light of your individual circumstances and expertise and contribute accordingly! For example if you have a certain type of income or receive a certain credit or deduction, and those are not yet implemented, we would gratefully assist you to implement them!
+
+## Pull Request Guidelines
+
+- Start all commits with a capital letter and imperitive mood, no period at the end.
+
+- Each commit first line should be max 50 characters, followed by a blank line, followed by as much detail as you think is needed, wrapped at 80 characters.
+
+- Github PR title style is the same as commit first line style.
+
+- It's OK to have multiple small commits as you work on the PR - we apply squash merges to all PRs at the end. In particular, you are free to rebase your commits so long as your pull request is in Draft status. As soon as you take it out of draft status, request a review, or receive a review on your pull request, please do not rebase or force push.
+
+- If adding new feature:
+
+  - Provide some reason to add this feature.
+
+- If fixing a bug:
+  - If you are resolving a special issue, add `Fixes: #xxx` (#xxx is the issue id) in your PR body. When your PR is merged, the maintainer will move this information into the final commit message for our release log.
+  - Provide detailed description of the bug in the PR, or link to an issue that does.
+
+## Development Guide
+
+### General Setup
+
+First, [join our Discord server](https://discord.gg/dAaz472mPz) and let us know that you want to contribute. This way we can point you in the right direction and help ensure your contribution will be as helpful as possible.
+
+To set up your machine for development, review the [Architecture doc](ARCHITECTURE.md), for required links to set up NPM > 7 and Rust.
+
+Next, fork and clone this repo. 
+
+Try 
+
+* `npm run desktop` to try out the desktop application. Setting the environment variable `BROWSER=none` will stop the web browser from also loading for you.
+* `npm run start` will serve the site locally on port 3000, so you can view it at `localhost:3000` in your browser.
+
+
+#### Stack
+
+This app uses React and Redux on typescript. For this reason the structure of this app may look familiar to you if you have used redux before, and it may seem opaque and confusing to you if you have not used Redux before. [Here is a great guide you can refer to about redux](https://redux.js.org/tutorials/fundamentals/part-1-overview)
```

Comment:
```suggestion
This app uses Typescript React and Redux. The structure of this app may look familiar to you if you have used redux before, and it may seem opaque and confusing to you if you have not used Redux before. [Here is a great guide you can refer to about redux](https://redux.js.org/tutorials/fundamentals/part-1-overview)
```